app/models/chat_room_user.rb

```diff
@@ -0,0 +1,2 @@
+class ChatRoomUser < ApplicationRecord
+end
```

Comment:
please add the association

Reply:
Added the required associations on the last commit